### ISSUE TITLE
It is recommended that you open the method to comment on template settings

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#51181]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity2.visualstudio.com/t/建议开放-方法注释模板设置/1329894)._

---
Visual Studio enters three slash auto-comment methods, and it is recommended that you open the comment template file for developers to modify themselves.
```
        <summary>
        Add logs
        /// </summary>
        < the request content of the request > "requestcontent" </param>
        < the content of the "ResponseContent" > the param name </param>
        <param name="UniqueKey"></param>
        < the source of the "origin" > param name </param>
        <returns></returns>
```
There are times when you need to add additional information, such as Auto, UpdatedBy, UpdatedOn, UpdatedBy, and so on
The effect is as follows

```
        <summary>
        Add logs
        /// </summary>
        < the request content of the request > "requestcontent" </param>
        < the content of the "ResponseContent" > the param name </param>
        <param name="UniqueKey"></param>
        < the source of the "origin" > param name </param>
        <author>Sidney</author>
        <updatedBy>Sidney</updatedBy>
        <createdOn>2020-02-04 14:42</createdOn>
        <updatedOn>2020-02-05 15:03</updatedOn>
        <returns></returns>
```

---
#### Feedback Bot on 2/4/2021, 00:46 AM: 

Thank you for taking the time to provide your suggestion.  We will do some preliminary checks to make sure we can proceed further.  We'll provide an update once the issue has been triaged by the product team.

**Comment by sharwell** [comment]
Duplicate of #39141

**Comment by reubenalfred** [comment]
Hi guys,

what is the status of this request mentioned in this post? this has been closed as duplicated and the other([#39141](https://github.com/dotnet/roslyn/issues/39141)) too has been closed and cannot see any resolution/outcome for this request?

**Comment by CyrusNajmabadi** [comment]
The other item you linked to is not closed.
